tests/README.md

```diff
@@ -15,3 +15,26 @@
 # the `token` directory
 1. Run `node mint-rio.js ropsten <metamask address>` where `metamask address` is your MetaMask wallet address after typing 'ropsten' to mint RIO token to wallet.
 2. Run `node node mint.js bsctestnet <metamask address>` where `metamask address` is your MetaMask wallet address to mint tToken to wallet.
+
```

Comment:
The above content now is not properly, it was for old testing but currently we have refined to use deposit_withdraw_test.ts and gen_input, etc. We do not use test_util.ts anymore.

Please check the old README content and keep the correct/useful texts. and remove those  unnecessary  texts